Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -309,6 +320,24 @@ private void MenuFlyout_Opened(object sender, object e)
             IsOpened = true;
             VisualStateManager.GoToState(this, "Opened", true);
             _parentMenu.IsInTransitionState = false;
+
+            if (!_menuFlyoutRepositioned)
```

Comment:
Yes, the first time when checking if it should be re positioned, and the second time if it's being shown re positioned